builds/e2e/templates/longhaul-setup.yaml

```diff
@@ -0,0 +1,90 @@
+parameters:
+  edgelet.artifact.name: ''
+  images.artifact.name: ''
+  aziotis.artifact.name: ''
+  aziotis.package.filter: ''
+  quickstart.artifactName: ''
+
+steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 100
+    submodules: recursive
+  - task: AzureKeyVault@1
+    displayName: 'Azure Key Vault: EdgeBuildkv'
+    inputs:
+      azureSubscription: $(azure.subscription)
+      KeyVaultName: 'edgebuildkv'
+      SecretsFilter: >- 
+        edgebuilds-azurecr-io-username,
+        edgebuilds-azurecr-io-pwd,
+        StorageAccountMasterKeyStress,
+        SnitchLongHaulAlertUrl,
+        kvLogAnalyticWorkspaceId,
+        kvLogAnalyticSharedKey,
+        EdgeLonghaulStorageAccountConnString,
+        GitHubAccessToken
+  - task: AzureKeyVault@1
+    displayName: 'Azure Key Vault: $(azure.keyVault)'
+    inputs:
+      azureSubscription: $(azure.subscription)
+      KeyVaultName: $(azure.keyVault)
+      SecretsFilter: >- 
+        IotHub-ConnStr,
+        IotHub-EventHubConnStr
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download Edgelet Artifacts'
+    inputs:
+      buildType: specific
+      project: $(vsts.project)
+      pipeline: $(edgelet.package.build)
+      branchName: $(edgelet.package.branchName)
+      buildVersionToDownload: latestFromBranch
+      downloadPath: '$(Build.StagingDirectory)'
+      artifactName: ${{ parameters['edgelet.artifact.name'] }}
+      allowPartiallySucceededBuilds: true
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download Images Artifacts'
+    inputs:
+      buildType: specific
+      project: $(vsts.project)
+      pipeline: $(images.build)
+      branchName: $(images.branchName)
+      buildVersionToDownload: latestFromBranch
```

Comment:
Umm... If the image from release/1.1 is the latest, I think it can cause the issue here.

Reply:
I didn't change anything here, but just moved this into its own yaml. 

It takes in the branch name as a parameter so it will not be possible to grab 1.1 bits from a 1.2 test run. Unless I am misunderstanding the concern?